app/views/playlists/_related_items.html.erb

```diff
@@ -22,9 +22,14 @@ Unless required by applicable law or agreed to in writing, software distributed
         <tr>
```

Comment:
Could we `render 'playlist_items/_related_items'` here instead of duplicating it?